tests/compilation/test_predictor_rl.py

```
@@ -22,6 +22,8 @@
 from qiskit.transpiler import InstructionProperties, Layout, Target, TranspileLayout
 from qiskit.transpiler.passes import GatesInBasis
 
+import mqt.predictor.rl.actions as actions_module
+import mqt.predictor.rl.actions.qiskit_actions as qiskit_actions
```

Ruff (0.15.17)

[warning] Use `from mqt.predictor.rl.actions import qiskit_actions` in lieu of alias

Replace with `from mqt.predictor.rl.actions import qiskit_actions`

(PLR0402)